src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenIfOperator.vb

```diff
@@ -15,6 +15,494 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.UnitTests
     Public Class CodeGenIfOperator
         Inherits BasicTestBase
 
+        <Fact, WorkItem(61483, "https://github.com/dotnet/roslyn/issues/61483")>
+        Public Sub Branchless_Compare()
+            Dim source =
+<compilation>
+    <file name="a.vb"><![CDATA[
+        Imports System.Console
+        Module C
+            Sub Main()
+                WriteLine(Comp(1, 2))
+                WriteLine(Comp(3, 3))
+                WriteLine(Comp(5, 4))
+            End Sub
+
+            Function Comp(x As Integer, y As Integer) As Integer
+                Dim tmp1 As Integer = If(x > y, 1, 0)
+                Dim tmp2 As Integer = If(x < y, 1, 0)
+                Return tmp1 - tmp2
+            End Function
+        End Module
+    ]]></file>
+</compilation>
+            Dim expectedOutput = <![CDATA[
+-1
+0
+1
+]]>
+            Dim verifier = CompileAndVerify(source, expectedOutput, options:=TestOptions.DebugExe)
+            verifier.VerifyDiagnostics()
+            verifier.VerifyIL("C.Comp", <![CDATA[
+{
+  // Code size       27 (0x1b)
+  .maxstack  2
+  .locals init (Integer V_0, //Comp
+                Integer V_1, //tmp1
+                Integer V_2) //tmp2
+  IL_0000:  nop
+  IL_0001:  ldarg.0
+  IL_0002:  ldarg.1
+  IL_0003:  bgt.s      IL_0008
+  IL_0005:  ldc.i4.0
+  IL_0006:  br.s       IL_0009
+  IL_0008:  ldc.i4.1
+  IL_0009:  stloc.1
+  IL_000a:  ldarg.0
+  IL_000b:  ldarg.1
+  IL_000c:  blt.s      IL_0011
+  IL_000e:  ldc.i4.0
+  IL_000f:  br.s       IL_0012
+  IL_0011:  ldc.i4.1
+  IL_0012:  stloc.2
+  IL_0013:  ldloc.1
+  IL_0014:  ldloc.2
+  IL_0015:  sub.ovf
+  IL_0016:  stloc.0
+  IL_0017:  br.s       IL_0019
+  IL_0019:  ldloc.0
+  IL_001a:  ret
+}
+]]>)
+            verifier = CompileAndVerify(source, expectedOutput, options:=TestOptions.ReleaseExe)
+            verifier.VerifyDiagnostics()
+            verifier.VerifyIL("C.Comp", <![CDATA[
+{
+  // Code size       12 (0xc)
+  .maxstack  3
+  .locals init (Integer V_0) //tmp2
+  IL_0000:  ldarg.0
+  IL_0001:  ldarg.1
+  IL_0002:  cgt
+  IL_0004:  ldarg.0
+  IL_0005:  ldarg.1
+  IL_0006:  clt
+  IL_0008:  stloc.0
+  IL_0009:  ldloc.0
+  IL_000a:  sub.ovf
+  IL_000b:  ret
+}
+]]>)
+        End Sub
+
+        <Fact, WorkItem(61483, "https://github.com/dotnet/roslyn/issues/61483")>
+        Public Sub Branchless_Operations()
```

Comment:
>Branchless_Operations [](http://example.com/codeflow?start=19&length=21)

Consider splitting up the test. It is really hard to map IL to specific VB code.

Reply:
Sorry, I should have used `VerifyMethodBody` which adds sequence points (as I do in the C# part). Changed - hopefully that helps and we don't need to split the test?